akka/src/main/scala/cakesolutions/kafka/akka/Offsets.scala

```diff
@@ -3,12 +3,24 @@ package cakesolutions.kafka.akka
 import org.apache.kafka.clients.consumer.OffsetAndMetadata
 import org.apache.kafka.common.TopicPartition
 
+/**
+  * Helper functions for [[Offsets]].
+  */
 object Offsets {
+
+  /**
+    * Offsets map with no offset information.
+    */
   def empty: Offsets = Offsets(Map.empty)
 }
 
 /**
   * Map of partitions to partition offsets.
+  *
+  * [[KafkaConsumerActor]] sends [[Offsets]] along with a batch of data related to the offsets.
+  * The offsets in this case represent the point at which the Kafka consumer reached after consuming the batch.
```

Comment:
The "point at which" bit doesn't read so well. It might be improved just by removing "at which".